roles/video_reserves/vars/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+# vars file for video_reserves
+vr_git_repo: "git@github.com:PrincetonUniversityLibrary/video_reserves.git"
+apache_doc_root: "/var/www/vr"
+server_name: "{{ ansible_hostname }}"
+app_db_host: "{{ db_host | default('localhost') }}"
+app_db_user: "{{ vr_db_user | default('vr_db_user') }}"
+app_db_password: "{{ vr_db_password | default('change_me') }}"
+app_db_port: "{{ vr_db_port | default('3306') }}"
```

Comment:
The code is currently hardcoded to the snakeoil cert: https://github.com/PrincetonUniversityLibrary/video_reserves/blob/0b5536ad637fc0b11189e342420b600a5c38f13b/hrc/vod/clip.php#L32  -- We'd like to create a real cert instead and update the code accordingly.